src/plugins/skills/src/systems/update_skill_combos.rs

```diff
@@ -204,14 +232,15 @@ mod tests {
 
 	#[test]
 	fn queue_not_marked_changed_when_non_added() {
-		let mut app = setup();
+		let (slots, items) = setup_slots([]);
+		let mut app = setup_app(items);
 		let entity = app
 			.world_mut()
 			.spawn((
 				Changed::<_Queue>::new(false),
 				_Combos::new(),
 				_Queue::default(),
-				Slots::default(),
+				slots,
```

Comment:
This makes sense in principle. However, if the asset is missing, then something has seriously gone wrong. As long as we hold a handle to an asset, bevy should never unload it. I am considering writing a kind of asset wrapper around bevies asset class, so that asset misses can be logged. But it is not clear, how we could solve such a behavior in bevy.

Can you create an issue, relating to this. It should express that a decision should be made how to handle failed asset retrievals, so we do not need a custom solution every time we try to get an asset through a handle.